m365-developer-proxy-plugins/MockResponses/MockResponsePlugin.cs

```diff
@@ -16,7 +16,9 @@
 namespace Microsoft365.DeveloperProxy.Plugins.MocksResponses;
 
 internal class MockResponseConfiguration {
+    [JsonIgnore]
     public bool NoMocks { get; set; } = false;
+    [JsonIgnore]
```

Comment:
won't this prevent passing the location of the mock file via config?

Reply:
The way I understood it, is that it's for serialization, but let me double check that it doesn't affect deserialization. Good catch

Reply:
Just verified and it doesn't prevent configuring the mocks file via config